src/http-client/index.ts

```diff
@@ -8,7 +9,7 @@ export { FetchClient } from "./fetch-client";
  */
 export type HTTPRequest = {
   data: QueryRequest;
-  headers: Record<string, string | string[]>;
+  headers: Record<string, string>;
```

Comment:
is this revert intentional?

Reply:
Yea, I applied the change before to request headers, but I mean to only update the headers type for response headers.